core/kazoo_attachments/src/kz_att_http.erl

```diff
@@ -101,8 +101,11 @@ fetch_attachment(HandlerProps, DbName, DocId, AName) ->
     case kz_json:get_value(<<"url">>, HandlerProps) of
         'undefined' -> kz_att_error:new('invalid_data', Routines);
         Url ->
-            AttSettings = kz_json:get_value(<<"handler_props">>, HandlerProps),
-            handle_fetch_attachment_resp(fetch_attachment(Url), AttSettings, Routines)
+            Base64Encoded = kz_json:is_true(<<"base64_encode_data">>, HandlerProps, 'false'),
```

Comment:
this is not needed for decoding, httpc will decode if needed and the server can send or not the attachment encoded, it does not depend on what we saved (still useful for debugging purposes)